user_guides/servlet/src/main/jbake/content/config.adoc

```diff
@@ -58,7 +58,7 @@ with the necessary configuration steps for your implementation and place
 the file under the `<TS_HOME>/bin/xml/impl/<your_impl>` directory.
 
 For more information, you may wish to view `<TS_HOME>/bin/xml/impl/glassfish/config.vi.xml`,
-the configuration file for Eclipse EE4J Jakarta EE 8 Compatible Implementation, Eclipse GlassFish.
+the configuration file for Eclipse EE4J Jakarta EE 9 Compatible Implementation, Eclipse GlassFish.
```

Comment:
Should we remove ```Eclipse EE4J```?  

FYI, I see around 65 references to ```Eclipse EE4J``` in the jakartaee-tck source and 3 references to just ```EE4J```.

Reply:
@arjantijms as per discussion on [Required Fixes on the naming of Jakarta EE across its mediums] It is NOT Eclipse Jakarta EE =/ It is Jakarta EE](https://www.eclipse.org/lists/jakarta.ee-community/msg02411.html), we should change "Eclipse EE4J Jakarta EE" to "Jakarta EE".

Could you please update your pull request to use "Jakarta EE 9".  Thank you!

Scott

Reply:
Thanks for the extra pair of eyes Scott, done!